05-UI-ReactJS/topic-3/src/App.js

```diff
@@ -35,18 +36,21 @@ class App extends React.Component {
       },
     ]});
   }
-  handleAddMovie(movie){
-    //data will not persist
-    let movies =this.state.movies;
-    movies.push(movie);
-    this.setState({movies:movies});
+  handleAddMovie(m){
```

Comment:
There is not issue on writting more, please make your code be descriptive. I'd write `movie` instead of `m`